base_app.htm

```diff
@@ -0,0 +1,60 @@
+<%inherit file="homepage/templates/base.htm" />
+<%! from catalog import models as cmod %>
+
+  <%block name="navitems">
+<ul class="nav navbar-nav navbar-right">
+    <li class="nav-item ${ 'active' if request.dmp.page == 'index' else ''}"><a class="nav-link" href="/">Home</a></li>
+    <li class="nav-item ${ 'active' if request.dmp.page == 'catalog/index' else ''}"><a class="nav-link" href="/catalog/index/">Catalog</a></li>
+    <li class="nav-item ${ 'active' if request.dmp.page == 'sections' else ''}"><a class="nav-link" href="/sections/">Sections</a></li>
+    <li class="nav-item ${ 'active' if request.dmp.page == 'about' else ''}"><a class="nav-link" href="/about/">About</a></li>
+    <li class="nav-item ${ 'active' if request.dmp.page == 'contact' else ''}"><a class="nav-link" href="/contact/">Contact</a></li>
+    <li class="nav-item ${ 'active' if request.dmp.page == 'terms' else ''}"><a class="nav-link" href="/terms/">Terms</a></li>
+    <li class="nav-item ${ 'active' if request.dmp.page == 'faq' else ''}"><a class="nav-link" href="/faq/">FAQ</a></li>
+% if request.user.is_authenticated:
+  <li><a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">${request.user.email} <span class="caret"></span></a>
+    <ul class="dropdown-menu">
+      <li><a class="nav-link" href="/account/index/">Account</a></li>
+      <li><a class="nav-link" href="/account/logout/">Log Out</a></li>
+    </ul>
+  </li>
+% else:
+<li><a href="#" class="dropdown-toggle" data-toggle="dropdown" role="button" aria-haspopup="true" aria-expanded="false">Log In<span class="caret"></span></a>
+  <ul class="dropdown-menu">
+    <li><a class="nav-link" href="/account/signup/">Sign Up</a></li>
+    <li><a class="nav-link" href="/account/login/">Log In</a></li>
+  </ul>
+</li>
+ % endif
+ </ul>
+</%block>
+
+<%block name="content_left">
+  <ul id="category_list">
+    <li class="${ 'activeLI' if category_name =='Products' else ''}"><a href="/catalog/index/" class="side-nav">All Products</a></li>
```

Comment:
interesting use of an active class here! I think I used a text style but this is a better idea :)

Reply:
thank you